independent-publisher-connectors/Strava/apiProperties.json

```diff
@@ -0,0 +1,35 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "76586",
```

Comment:
@RichardWie Please replace it with `"[[TEST]]"` or `"[[DUMMY]]"` as we do not want to expose client credentials to public.